website/src/controllers/api-v2/members.js

```diff
@@ -38,7 +38,7 @@ api.sendMessage = function(user, member, data){
     msg = data.message
   } else {
     msg = "`Hello " + member.profile.name + ", " + user.profile.name + " has sent you ";
-    msg += (data.type=='gems') ? data.gems.amount + " gems!`" : shared.content.subscriptionBlocks[data.subscription.key].months + " months of subscription!`";
+    msg += (data.type=='gems') ? data.gems.amount + " gem" + ((data.gems.amount>1)?"s":"") + "!`" : shared.content.subscriptionBlocks[data.subscription.key].months + " month" + ((shared.content.subscriptionBlocks[data.subscription.key].months>1)?"s":"") + " of subscription!`";
```

Comment:
Mostly we're using a multiple ternaries on this line, which makes it very very difficult to read. I'd advocate for a more verbose `if {} else {}` logic. IE:

```js
if (data.type === 'gems') {
  var gemAmount = data.gems.amount;
  var gemLabel = gemAmount > 1 ? "gems" : "gem";
  msg += gemAmount + " " + gemLabel + "!";
} else {
  var monthAmount = shared.content.subscriptionBlocks[data.subscription.key].months;
  var monthLabel = monthAmount > 1 ? 'months' : 'month';
  msg += months + " " + monthLabel + " of subscription!`"
}
```

IMO, that's much easier to parse than what we currently have.